src/components/inputs/rich-text-input/rich-text-input.js

```diff
@@ -9,17 +9,22 @@ import plugins from '../../internals/rich-text-plugins';
 import html from '../../internals/rich-text-utils/html';
 import isEmpty from '../../internals/rich-text-utils/is-empty';
 
-const propsToState = props => ({
-  serializedValue: props.value || '',
-  internalSlateValue: html.deserialize(props.value || ''),
-});
-
 class RichTextInput extends React.PureComponent {
-  state = propsToState(this.props);
+  serializedValue;
+  internalSlateValue;
+
+  constructor(props) {
+    super(props);
+    this.serializedValue = props.value || '';
+    this.internalSlateValue = html.deserialize(this.props.value || '');
+  }
```

Comment:
Why do we need a constructor and can't just `state = {}` as a property initializer?

Reply:
because these are not state, they are refs?

Reply:
🤔

Reply:
Sorry, I meant `internalSlateValue = props.value || ''` and ... not on state but outside of the constructor.

Reply:
oh yeah ok